dashboards/google-vertex-ai/vertex-ai-provisioned-throughput-utilization.json

```diff
@@ -0,0 +1,112 @@
+{
+  "displayName": "Provisioned Throughput Utilization",
+  "dashboardFilters": [],
+  "labels": {},
+  "mosaicLayout": {
+    "columns": 48,
+    "tiles": [
+      {
+        "height": 26,
+        "width": 48,
+        "widget": {
+          "title": "",
+          "id": "",
+          "xyChart": {
+            "chartOptions": {
+              "displayHorizontal": false,
+              "mode": "COLOR",
+              "showLegend": false
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "legendTemplate": "${resource.labels.model_user_id} ${resource.labels.location}${resource.labels.monitored_resource_container}",
```

Comment:
nit: can we add a space between `${resource.labels.location}` and `${resource.labels.monitored_resource_container}`